src/EventBuilder/Platforms/Tizen.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MS-PL license.
+// See the LICENSE file in the project root for more information.
+
+using NuGet;
+using Polly;
+using Serilog;
+using System;
+using System.IO;
+using System.Linq;
+
+namespace EventBuilder.Platforms
+{
+    public class Tizen : BasePlatform
+    {
+        private const string _packageName = "Tizen.NET";
+
+        public Tizen()
+        {
+            var packageUnzipPath = Environment.CurrentDirectory;
+
+            Log.Debug("Package unzip path is {PackageUnzipPath}", packageUnzipPath);
+
+            var retryPolicy = Policy
+                .Handle<Exception>()
+                .WaitAndRetry(
+                    5,
+                    retryAttempt => TimeSpan.FromSeconds(Math.Pow(2, retryAttempt)),
+                    (exception, timeSpan, context) => {
+                        Log.Warning(
+                            "An exception was thrown whilst retrieving or installing {packageName}: {exception}",
+                            _packageName, exception);
```

Comment:
The string interpolations here (and ln22 and ln42) need a $ at the start of the string, e.g. ln 58.